Classes/Section Controllers/IGListSectionController+Utils.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  IGListSectionController+Utils.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 11/26/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+extension ListSectionController {
+
+    var util_context: ListCollectionContext {
+        guard let context = collectionContext else {
+            fatalError("Missing context")
+        }
+        return context
+    }
+
+    func util_dequeueCell<T: AnyObject>(index: Int) -> T {
```

Comment:
Maybe keep the naming consistent here (`dequeueReusableCell`) for easier lookup via autocompletion? Also not sold on the `util_`, think it's pretty hard to discover atm 😕